scripts/read_and_validate_logs.sh

```diff
@@ -20,4 +21,20 @@ if [ -z "$STREAM_NAME" ] || [ "$STREAM_NAME" = "null" ]; then
     exit 1
 fi
 
-$AWSLOCAL logs get-log-events --log-group-name /aws/lambda/$EXAMPLE --log-stream-name "$STREAM_NAME" --start-from-head | jq -r '.events[].message'
+export LOGS=$($AWSLOCAL logs get-log-events --log-group-name /aws/lambda/$EXAMPLE --log-stream-name "$STREAM_NAME" --start-from-head \
+ | jq -r '.events[].message')
+
+log=$(echo $LOGS | sed -e 's/.*LATEST\(.*\)END.*/\1/' |  sed 's/\x1B\[[0-9;]\{1,\}[A-Za-z]//g')
+if [ -z "$log" ] || [ "$log" = "null" ]; then
+  echo "Empty logs"
+  exit 1
+fi
+
+message=$(echo $message | jq -r ".fields.message")
+if [ "$message" = "$EXPECTED_MESSAGE" ]; then
+  echo "Test passed!"
```

Comment:
```suggestion
  echo "Found expected message: $message"
  echo "Test passed!"
```

Since these scripts are pretty chatty anyway, I think it would be worthwhile echoing the actual message that was found here. It'll make it a bit easier to visually confirm that the test itself is working as expected.